constants.json

```diff
@@ -1,98 +1,92 @@
 {
-    "jbide_master_versions": {
-	"function": "listVersions",
-	"project": "JBIDE",
-	"pattern": "4.5.(?!x).*"
+	"jbide_master_versions": {
+		"function": "listVersions",
+		"project": "JBIDE",
+		"pattern": "4.19.1.(?!x).*"
```

Comment:
Not sure to understand the pattern syntax but I would expect that the .1 would be replacing the .(?!x) ?

Reply:
pattern is to exclude any .x versions. should works without it, too.